ui-src/src/cells/holo-vault/components/profile/profile.tsx

```diff
@@ -56,6 +56,8 @@ export interface StateProps {
 
 export interface State {
   profile: ProfileType
+  selectedPersona: PersonaType
+  personas: Array<PersonaType>
```

Comment:
Again, from what I have read copying props directly to the state is a bit of an anti-pattern. From a quick look at this file it seems like we could just use the copy of personas in the props? Or have I missed something

Reply:
When we change the selected Persona the list gets reordered with the selected one first. I'm pretty sure we need personas in state so the UI rerenders. Happy to look at it again though.

Reply:
Ah yeah that's right. I guess we can keep it for now but definitely something to look at in a refactor later